applications/matter_bridge/doc/adding_bridged_matter_device.rst

```diff
@@ -0,0 +1,397 @@
+
+The Matter Bridge application supports bridging only a few Matter device types due to practical reasons.
+You can select any of the available :ref:`Matter device types <ug_matter_device_types>` and add support to it in the application.
+In this guide the ``Pressure Sensor`` device type was presented as an example.
+
+Based on :ref:`Matter Bridge architecture <ug_matter_overview_bridge_ncs_implementation>` you will need to implement the ``Matter Bridged Device`` and ``Bridged Device Data Provider`` roles for the newly added Matter device type.
+The Matter Bridge application supports simulated and Bluetooth LE Bridged Device Data Providers.
+In this guide the simulated provider example was presented, but the process looks similar for Bluetooth LE provider as well.
+
+Complete the following steps to add the ``Pressure Sensor`` device type support to the Matter Bridge.
+
+1. Enable ``Pressure Measurement`` cluster for the endpoint ``2`` in the :file:`src/bridge.zap` file and re-generate files located in the :file:`src/zap-generated` directory.
+   To learn how to modify the :file:`.zap` file and re-generate the :file:`zap-generated` directory, see the :ref:`ug_matter_creating_accessory_edit_zap` in the :ref:`ug_matter_creating_accessory` user guide.
+#. Implement the ``Matter Bridged Device`` role.
+
+   a. Create :file:`pressure_sensor.cpp` and :file:`pressure_sensor.h` files in the :file:`src/bridged_device_types` directory.
+   #. Open the :file:`nrf/samples/matter/common/bridge/matter_bridged_device.h` to see :c:struct:`MatterBridgedDevice` class constructor.
+   #. Add new :c:struct:`PressureSensorDevice` class inherting from the :c:struct:`MatterBridgedDevice` and implement its constructor in the :file:`pressure_sensor.cpp` and :file:`pressure_sensor.h` files.
+
+      - :file:`pressure_sensor.h`
+
+         .. code-block:: C++
+
+            #pragma once
+
+            #include "matter_bridged_device.h"
+
+            class PressureSensorDevice : public MatterBridgedDevice {
+            public:
+
+            PressureSensorDevice(const char *nodeLabel);
+            static constexpr uint16_t kPressureSensorDeviceTypeId = 0x0305;
+
+            };
+
+      - :file:`pressure_sensor.cpp`
+
+         .. code-block:: C++
+
+            #include "pressure_sensor.h"
+
+            PressureSensorDevice::PressureSensorDevice(const char *nodeLabel) : MatterBridgedDevice(nodeLabel) {}
+
+   #. Declare all mandatory clusters for the ``Pressure Sensor`` device type and fill the appropriate :c:struct:`MatterBridgedDevice` class fields in the :c:struct:`PressureSensorDevice` class constructor.
+      The ``Pressure Sensor`` device requires ``Descriptor``, ``Bridged Device Basic Information`` and ``Identify`` clusters that can be declared using helper macros from the :file:`nrf/samples/matter/common/bridge/matter_bridged_device.h` and ``Pressure Measurement`` cluster that has to be described in the application.
```

Comment:
Well, we could do that, but I'm not sure if it's really needed, because we present an example below that precisely shows which macros are used, there is also API documentation for them and their names are sort of self-explanatory. My point was to only mention our custom macros for mandatory clusters, because it's not obvious why we use them.